scripts/docgen/api/findUnusedExports.ts

```diff
@@ -176,13 +203,23 @@ export const getPublicApiExports = (project: Project): ApiExports[] => {
     for (const publicApi of publicApis) {
         const packageRoot = join(rootDir, publicApi)
         const packageJsonData: PackageJson = readPackageJson(packageRoot)
-        const entryPoints = getEntryPointsToRelativeDtsPaths(packageJsonData)
-        const pathToSourceFile = join(packageRoot, ...entryPoints[0])
-        const sourceFile = project.addSourceFileAtPath(pathToSourceFile)
+        try {
+            const entryPoints =
+                getEntryPointsToRelativeDtsPaths(packageJsonData)
+            const pathToSourceFile = join(packageRoot, ...entryPoints[0])
+            const sourceFile = project.addSourceFileAtPath(pathToSourceFile)
 
-        const entryPointExports = getEntryPointExports(sourceFile, packageRoot)
-        if (entryPointExports.length) {
-            apiExports.push(...entryPointExports)
+            const entryPointExports = getEntryPointExports(
+                sourceFile,
+                packageRoot
+            )
+            if (entryPointExports.length) {
+                apiExports.push(...entryPointExports)
+            }
+        } catch (e) {
+            throw new Error(
```

Comment:
Let's remove this try block and just let the error bubble up naturally since it seems like we're not really adding any particular context other than a possible assumption.